python/perspective/setup.py

```diff
@@ -139,13 +139,8 @@ def build_extension_cmake(self, ext):
             "-DPSP_WASM_BUILD=0",
             "-DPSP_PYTHON_BUILD=1",
             "-DPSP_PYTHON_VERSION={}".format(PYTHON_VERSION),
-            "-DPython_ADDITIONAL_VERSIONS={}".format(PYTHON_VERSION),
```

Comment:
are we sure that these are all not necessary? Did you test under a virtualenv

https://github.com/finos/perspective/commit/61557c6a28a0c70e00cd4ee9160ef22e65c2ad99